racingcar-java/src/main/java/step3/domain/Round.java

```diff
@@ -0,0 +1,24 @@
+package step3.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Round {
+
+    private Map<Integer, List<Car>> round = new HashMap<>();
```

Comment:
```suggestion
    private final Map<Integer, List<Car>> round = new HashMap<>();
```
로 변경한다면 round에 대한 변경을 막을 수 있겠네요.

Reply:
아 그렇네요 제가 생각이 조금 짧았던 것 같습니다 감사합니다 :)